lib/main.dart

```diff
@@ -22,30 +22,22 @@ void main() {
 }
 
 void _buildDependencies() {
-  StarportApp.networkInfo = NetworkInfo(
-    bech32Hrp: 'cosmos',
-    lcdInfo: LCDInfo(
-        host: Platform.isAndroid ? 'http://10.0.2.2' : 'http://localhost'),
-    grpcInfo: GRPCInfo(
-        host: Platform.isAndroid ? 'http://10.0.2.2' : 'http://localhost'),
-  );
-
```

Comment:
this was causing the code to always try to access locally served chain instead of cosmos testnet, despite the `--dart-define` params

Reply:
Ahh i seeee